File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex17.java
//Importando Scanner
import java.util.Scanner;

public class Ex17 {
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Verificação de Senha\n\n");
		
		//vars
		int senha;
		
		//Entrada
		System.out.print("Insira a senha para acessar o sistema: ");
		senha = ent.nextInt();
		
		//Processamento
		if ( senha == 4531 )
			System.out.println("Acesso Permitido!\n");
		else
			System.out.println("Acesso Negado!\n");
		
	}

}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex17.c
#include <stdio.h>

main()
{
    printf("Validacao de Senha\n\n");

    //vars
    int senha;


    //Entrada
    printf("Por favor, insira a senha para acessar o programa: ");
    scanf("%i", &senha);

    if ( senha == 4531 )
        printf("Acesso permitido.\nBem-vindo!");
    else
        printf("Acesso negado");

}
